libr/anal/class.c

```diff
@@ -1238,21 +1238,22 @@ R_API RGraph *r_anal_class_get_inheritance_graph(RAnal *anal) {
 		r_graph_free (class_graph);
 		return NULL;
 	}
-	// O(1) lookup cache for processed nodes
-	// to lookup parents to create edges
+	/* O(1) lookup cache for processed nodes */
```

Comment:
why that? // is fine for single line comments

Reply:
No specific reason I just thought it looks better

Reply:
Imho they should stay as //

Reply:
Sure, done